src/main/java/ch/uzh/ifi/hase/soprafs24/websocket/events/GameChangesEvent.java

```diff
@@ -1,5 +1,7 @@
 package ch.uzh.ifi.hase.soprafs24.websocket.events;
 
+import ch.uzh.ifi.hase.soprafs24.controller.GameController;
+import ch.uzh.ifi.hase.soprafs24.model.game.Game;
 import ch.uzh.ifi.hase.soprafs24.websocket.dto.WSGameChangesDto;
 import lombok.Getter;
 import lombok.Setter;
```

Comment:
Could you eventually squash all commits? I don't think we need to add 25 commits to the history :)